lib/flapjack/gateways/jsonapi/metrics_methods.rb

```diff
@@ -30,10 +30,16 @@ def event_queue_length
             redis.llen('events')
           end
 
+          def event_queue
+            { 'length' => redis.llen('events') }
+          end
+
           def processed_events
```

Comment:
This and the next method seem like they should be put together, unless you're intending to add more parameters here at some point in future.